다리를지나는트럭.py

```diff
@@ -0,0 +1,19 @@
+# 프로그래머스: 스택/큐 다리를 지나는 트럭: https://programmers.co.kr/learn/courses/30/lessons/42583
+
+def solution(bridge_length, weight, truck_weights):
+    answer = 0
+    bridge = [0 for _ in range(bridge_length)]
```

Comment:
값(primitive type)의 복사하여 리스트를 만드는 경우 다음과 같이 단축할 수 있습니다. 
```suggestion
    bridge = [0] * bridge_length
```

단 리스트 처럼 주소값을 같는 타입의 경우 각 요소가 주소를 공유하게 되기 떄문에 그렇게 하면 안됩니다.
```python
bridge = [[]] * 10 # (X)
```